libs/common/src/lib/widgets/widget-generator.component.ts

```diff
@@ -279,6 +281,7 @@ export class WidgetGeneratorComponent implements OnInit, OnDestroy {
       .subscribe(([schema, config]) => {
         this.generativeModels = schema['generative_model']?.options || [];
         this.defaultModelFromSettings = config['generative_model'] || '';
+        this.defaultPromptFromSettings = config['user_prompts']?.[this.defaultModelFromSettings]?.['prompt'] || '';
```

Comment:
Maybe there's a problem in this here.
It seems like the models keys are not very consistent in the configuration.
For instance: `config['generative_model']` is `claude-3`
but the user prompt is stored in `config['user_prompts']['claude3']`

Reply:
good catch!
fixed